### ISSUE TITLE
[T7][Accessibility][Functional] –Procedure button doesn’t read on hover

### ISSUE THREAD

**Issue by Channamma** [Mohanme13/Feb-19-Bug-bash#173]
Procedure button doesn't read on hover
